pymc4/variational/updates.py

```diff
@@ -0,0 +1,108 @@
+"""Optimizers for ELBO convergence.
+
+These optimizers wrap tf.optimizers with defaults from PyMC3.
```

Comment:
add and explanation that optimizers API is too transparent that you can use plain to optimizers